crypto/hkdf/src/lib.rs

```diff
@@ -110,9 +110,10 @@ impl<H: BitcoinHash> Hkdf<H> {
 
 #[cfg(test)]
 mod tests {
-    use crate::Hkdf;
     use bitcoin_hashes::Hash as BitcoinHash;
 
+    use crate::Hkdf;
```

Comment:
Sorry about that, rebasing seems to have skipped commit hooks somehow

Reply:
I have a `rustfmt` on file save set, so I don't even notice while formatting happens. :)